test/test_limits.py

```diff
@@ -37,9 +38,32 @@ async def test_pool_size(pg, bouncer):
 
 @pytest.mark.asyncio
 async def test_min_pool_size(pg, bouncer):
-    bouncer.admin("set min_pool_size = 3")
```

Comment:
The test has been changed to not set the global `min_pool_size` anymore. Caused too many connection creations in a number of pools overloading the github test runner. 
Rather this test check various combinations with individual pools.